Mathlib/NumberTheory/DirichletCharacter/Basic.lean

```diff
@@ -182,6 +182,16 @@ lemma conductor_eq_zero_iff_level_eq_zero : conductor χ = 0 ↔ n = 0 := by
   rintro rfl
   exact Nat.sInf_eq_zero.mpr <| Or.inl <| level_mem_conductorSet χ
 
+lemma mem_conductorSet_eq_conductor {d : ℕ} (hd : d ∈ conductorSet χ) :
```

Comment:
The name of this lemma does not match the statement, Maybe `conductor_le_conductor_mem_conductorSet`?

Reply:
yes, done